### DISCUSSION TITLE
One sample in each domain

### DISCUSSION THREAD

**Discussion by mhzhang96** [jindongwang/transferlearning#372] [Q&A]
您好，我用EEG数据做domain generalization的时候，我的每一个human participants是一个domain，因此我的每个domain里的n_samples=1，在get_img_dataloader(args)里split成train和test的时候就会出错。
但我的data shape是（480，150，128），虽然n_subject=1但是n_samples可以是480，请问这个情况我该怎么处理呢？

**Comment by jindongwang** [comment]
那说明你需要把一个人的数据分成多条。一条是没法深度学习的呀。
